src/callbacks/onPreprocess/flattenData/getMaxValues.js

```diff
@@ -47,15 +46,19 @@ export default function getMaxValues(d) {
             } else {
                 //see if all selected config.plot_day was while participant was enrolled
                 var first = participant_obj[mKey + '_raw'][0];
-                var last = participant_obj[mKey + '_raw'].pop();
-                var before = config.plot_day < first.day;
-                var after = config.plot_day > last.day;
-                participant_obj.outOfRange = participant_obj.outOfRange || before || after;
+                var last = participant_obj[mKey + '_raw'].slice().pop(); // Array.pop() alters the original array, but not if you slice and dice it!
+
+                if ([config.x.column, config.y.column].includes(mKey)) {
+                    // out-of-range should be calculated study day of with x- and y-axis measures
```

Comment:
I've been debating whether we should include a size variable (if any) or perhaps just use all key measures here ...

Reply:
Line 105 uses all key measures. Leaning towards implementing that here as well.